src/garage/experiment/local_tf_runner.py

```diff
@@ -206,16 +213,17 @@ def save(self, epoch, paths=None):
         if paths:
             params['paths'] = paths
         params['last_epoch'] = epoch
-        snapshotter.save_itr_params(epoch, params)
+        self.snapshotter.save_itr_params(epoch, params)
 
         logger.log('Saved')
 
-    def restore(self, snapshot_dir, from_epoch='last'):
+    def restore(self, resume_from_dir, resume_from_epoch='last'):
         """Restore experiment from snapshot.
 
         Args:
-            snapshot_dir(str): Directory of snapshot.
-            from_epoch(str or int): The epoch to restore from.
+            resume_from_dir (str): Directory of the pickle file
+                to resume experiment from.
+            resume_from_epoch (str or int): The epoch to restore from.
```

Comment:
how about -1 for last? this is consistent with the rest of python.

Reply:
Are you suggesting we should let users do `from_epoch=0, from_epoch=-1` instead of `from_epoch='first', from_epoch='last'`?